utils/sampler/weighted_benchmark_test.go

```diff
@@ -119,14 +113,14 @@ func WeightedPowBenchmarkSampler(
 
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		_, _ = s.Sample(uint64(globalRNG.Int63n(int64(totalWeight))))
+		_, _ = s.Sample(globalRNG.Uint64n(totalWeight - 1))
```

Comment:
Why does
```
s.Sample(uint64(globalRNG.Int63n(int64(totalWeight))))
```
become 
```
s.Sample(globalRNG.Uint64n(totalWeight - 1))
```
here but below the analogous call
```
s.Sample(uint64(globalRNG.Int63n(math.MaxInt64)))
```
becomes 
```
s.Sample(globalRNG.Uint64n(math.MaxUint64))
```
i.e. not with a `-1`?

Reply:
:eagle: :eye: